src/backend/gl/src/device.rs

```diff
@@ -1088,6 +1088,10 @@ impl d::Device<B> for Device {
         self.create_swapchain_impl(surface, config)
     }
 
+    fn destroy_swapchain(&self, _swapchain: Swapchain) {
+        unimplemented!()
```

Comment:
let's keep the example running on GL

Reply:
It is not possible to get the example to run on GL even before this, as almost all destroy functions for the GL implementation is unimplemented